jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -228,6 +231,13 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
         return this;
       }
       if (context.isSampled()) {
+        if (tracer.isExpandExceptionLogs()) {
+          Map<String, Object> errorFields = exceptionLogs(fields);
```

Comment:
minor point, but couldn't we move all the logic into exceptionLogs() so that here we only have
```
fields = exceptionLogs(fields);
```
?

Internally it could do `Map<String, Object> errorFields = new HashMap<String, Object>(fields);`

Reply:
I have changed it to  ^^. I will merge or green